diff --git a/Makefile b/Makefile
index 445ddd7..9fb5745 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ DOCKERFILE := $(shell bash -c 'source hack/make/.detect-daemon-osarch && echo $$
 # env vars passed through directly to Docker's build scripts
 # to allow things like `make KEEPBUNDLE=1 binary` easily
 # `docs/sources/contributing/devenvironment.md ` and `project/PACKAGERS.md` have some limited documentation of some of these
-DOCKER_ENVS := \
+DOCKER_ENVS := -e DOCKER_GITCOMMIT \
 	-e BUILDFLAGS \
 	-e KEEPBUNDLE \
 	-e DOCKER_BUILD_GOGC \
